learning/tour-of-beam/frontend/lib/pages/tour/screen.dart

```diff
@@ -0,0 +1,358 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'package:easy_localization/easy_localization.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:playground_components/playground_components.dart';
+
+import '../../components/expansion_tile_wrapper.dart';
+import '../../components/filler_text.dart';
+import '../../components/scaffold.dart';
+import '../../constants/sizes.dart';
+import '../../generated/assets.gen.dart';
+import 'playground_demo.dart';
+
+class TourScreen extends StatelessWidget {
+  const TourScreen();
+
+  @override
+  Widget build(BuildContext context) {
+    return TobScaffold(
+      child: MediaQuery.of(context).size.width > ScreenBreakpoints.twoColumns
+          ? const _WideTour()
+          : const _NarrowTour(),
+    );
+  }
+}
+
+class _WideTour extends StatelessWidget {
+  const _WideTour();
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: const [
+        _ContentTree(),
+        Expanded(
+          child: SplitView(
+            direction: Axis.horizontal,
+            first: _Content(),
+            second: PlaygroundDemoWidget(),
+          ),
+        ),
+      ],
+    );
+  }
+}
+
+class _NarrowTour extends StatelessWidget {
+  const _NarrowTour();
+
+  @override
+  Widget build(BuildContext context) {
+    return SingleChildScrollView(
+      child: Column(
+        children: [
+          Row(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: const [
+              _ContentTree(),
+              Expanded(child: _Content()),
+            ],
+          ),
+          DecoratedBox(
+            decoration: BoxDecoration(
+              border: Border(
+                top: BorderSide(color: Theme.of(context).dividerColor),
+              ),
+            ),
+            child: const _NarrowScreenPlayground(),
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+class _ContentTree extends StatelessWidget {
+  const _ContentTree();
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      width: 250,
```

Comment:
Should be extracted as `_contentTreeWidth` constant.

Reply:
I will let you continue with this file once it is merged.